dvc/command/repro.py

```diff
@@ -136,7 +136,7 @@ def add_parser(subparsers, parent_parser):
         help="Reproduce all stages in the specified directory.",
     )
     repro_parser.add_argument(
-        "--ignore-build-cache",
+        "--ignore-run-cache",
         action="store_true",
         default=False,
         help="Reproduce all descendants of a changed stage even if their "
```

Comment:
does this description stay?

Reply:
Yes, but for `--force-downstream`. The new name reflects the functionality a lot better.

Reply:
Yep 👍